src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,160 @@
+      return 1;
+    }
+
+    if (bValue === null || bValue === undefined) {
+      return -1;
+    }
+
+    if (aValue < bValue) {
+      return sortOrder === 'asc' ? -1 : 1;
+    }
+
+    if (aValue > bValue) {
+      return sortOrder === 'asc' ? 1 : -1;
+    }
+
+    return 0;
+  });
+
+  const sortedIcon = (category: 'name' | 'sex' | 'born' | 'died') => {
+    if (searchParams.get('sort') === category) {
+      return searchParams.get('order') === 'desc'
+        ? 'fa-sort-down'
+        : 'fa-sort-up';
+    }
+
+    return 'fa-sort';
+  };
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>
+            <span className="is-flex is-flex-wrap-nowrap">
+              Name
+              <a
+                href="#/people?sort=name"
+                onClick={event => {
+                  event.preventDefault();
+                  handleSort('name');
+                }}
+              >
+                <span className="icon">
+                  <i className={cn('fas', sortedIcon('name'))} />
+                </span>
+              </a>
+            </span>
+          </th>
+
+          <th>
+            <span className="is-flex is-flex-wrap-nowrap">
+              Sex
+              <a
+                href="#/people?sort=sex"
+                onClick={event => {
+                  event.preventDefault();
+                  handleSort('sex');
+                }}
+              >
+                <span className="icon">
+                  <i className={cn('fas', sortedIcon('sex'))} />
+                </span>
+              </a>
+            </span>
+          </th>
+
+          <th>
+            <span className="is-flex is-flex-wrap-nowrap">
+              Born
+              <a
+                href="#/people?sort=born&amp;order=desc"
+                onClick={event => {
+                  event.preventDefault();
+                  handleSort('born');
+                }}
+              >
+                <span className="icon">
+                  <i className={cn('fas', sortedIcon('born'))} />
+                </span>
+              </a>
+            </span>
+          </th>
+
+          <th>
+            <span className="is-flex is-flex-wrap-nowrap">
+              Died
+              <a
+                href="#/people?sort=died"
+                onClick={event => {
+                  event.preventDefault();
+                  handleSort('died');
+                }}
+              >
+                <span className="icon">
+                  <i className={cn('fas', sortedIcon('died'))} />
+                </span>
+              </a>
+            </span>
+          </th>
+
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {sortedPeople.map(person => (
+          <PersonLink key={person.slug} person={person} people={people} />
```

Comment:
Ensure that the `PersonLink` component correctly handles the `people` prop, as it might be more efficient to pass only the necessary data rather than the entire list of people.